app/views/layouts/_navbar.html.erb

```diff
@@ -121,8 +153,8 @@
           <% end %>
         <% end %>
 
-        <% if current_user && current_user.administrator? %>
-          <li><%= link_to "<i class='material-icons left'>lock</i> Manage Autolab".html_safe, admin_path, title: "Perform system-wide administrative actions" %></li>
+        <% if current_user&.administrator? %>
+          <li><a class="dropdown-trigger" href="#!" data-target="dropdown3" data-beloworigin="true" data-hover="false"><i class='material-icons left'>lock</i>Manage Autolab<i class="material-icons right">arrow_drop_down</i></a></li>
         <% end %>
```

Comment:
Unaddressed comment from Joey:

Can you update your PR so that it removes any RSpec tests for admins/show?

Reply:
See `/spec/controllers/admins_controller_spec.rb`